source/includes/steps-install-shell-macos-homebrew.yaml

```diff
@@ -0,0 +1,37 @@
+---
+title: "Install Homebrew."
+ref: install-homebrew
+level: 4
+content: |
+
+   Run the following command to install the
+   `Homebrew <https://brew.sh/>`__ package manager:
+
+   .. code-block:: sh
+  
+      /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
```

Comment:
We should not be providing this step, but should instead link to the Homebrew instructions themselves:
- If homebrew changes this syntax, we have to remember to change this to match
- If a user is following MDB instructions, but runs into an issue specifically with the Homebrew install step that we provide, they will expect us to provide troubleshooting steps for this step. If we instead link to the Homebrew install steps, users are on the Homebrew page when they run into their issues, and are in the right place to ask for support, or click sidebar links to the Homebrew troubleshooting guide (i.e. read https://docs.brew.sh/ not https://docs.mongodb.com)

Reply:
Agreed. Updating.